pep-0570.rst

```diff
@@ -221,7 +327,7 @@ remains the same as the one for positional-or-keyword. Once
 a positional-only argument is provided with a default,
```

Comment:
In the folded portion of the diff (line 219/325), "The logic around optional values for positional-only argument" should either be:

"The logic around optional values for positional-only arguments"

or

"The logic around optional values for a positional-only argument"

I prefer the first one.

Also "remains the same as the one for positional-or-keyword."

should be

"remains the same as for positional-or-keyword arguments"

or

"remains the same as for a positional-or-keyword argument"